package.json

```
@@ -14,7 +14,8 @@
   ],
   "scripts": {
     "build": "gulp build",
-    "test": "gulp test"
+    "test": "gulp test",
+    "lint": "eslint . --fix"
```

Great!
But,
running `--fix` will give a different behaviour in CI (which might not be same as in local), we just want to check lint and dev can perform --fix locally!
```suggestion
    "lint": "eslint ."
```